.github/workflows/run-tests.yml

```diff
@@ -1,18 +1,17 @@
+---
 # See more at: https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions
 name: run-go-tests
 
 env:
-  GO111MODULE: on
+  GO111MODULE: "on"
 
-on:
+"on":
```

Comment:
Why in `"`?

Reply:
Because yamllint treats `on` as an error.
`warning  truthy value should be one of [false, true]  (truthy)`
https://github.com/adrienverge/yamllint/issues/430
There are two approaches: 1. Wrap on with `"`. 2. Add `# yamllint disable-line rule:truthy`